src/main/java/com/cloudbees/jenkins/support/actions/SupportAbstractItemAction.java

```diff
@@ -22,7 +22,7 @@ public SupportAbstractItemAction(AbstractItem target) {
 
     @Override
     public String getDisplayName() {
-        return Messages.SupportItemAction_DisplayName();
+        return shouldDisplay() ? Messages.SupportItemAction_DisplayName() : null;
```

Comment:
Wouldn't it be possible to implement methods in the super class to avoid having to change every method in the subclasses?

Reply:
Functionally what you suggest is possible (all action display names are the same) but they are in fact referring to a different message, eg:

https://github.com/jenkinsci/support-core-plugin/blob/09dbc25ed1f15000477fc62fd6b730085ca70049/src/main/resources/com/cloudbees/jenkins/support/Messages.properties#L25

https://github.com/jenkinsci/support-core-plugin/blob/4a3c12403766f8e28386df1d2bd9757e3ec2fae7/src/main/resources/com/cloudbees/jenkins/support/actions/Messages.properties#L25

I suppose I could build the key to retrieve the message from the class name, but maybe it's worth considering removing the duplication of messages instead. AFAIU, the name will always be Support and the icon will always be the same too. Wdyt?

Reply:
If the user result is the same, +1 for removing the duplication.

Reply:
Messages are made unique by #239